keyboards/ckeys/handwire_101/handwire_101.h

```diff
@@ -0,0 +1,23 @@
+#ifndef HANDWIRE_101_H
+#define HANDWIRE_101_H
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+// The following is an example using the Planck MIT layout
+// The first section contains all of the arguements
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
```

Comment:
Please change this to:

```suggestion
#define LAYOUT( \
```

QMK has settled on the term *layout* to describe the physical layout of the keyboard (how many keys, their sizes and positions, etc.) and *keymap* to describe what happens when a key is pressed. That being the case, the above usage is deprecated.